### PR TITLE
feat: pagination, update examples

### PR DESCRIPTION
**Pull request by sabineschaller**
Adds parameters for pagination on list endpoints
Updates examples

### CODE REVIEW

File: open-api-spec.yaml [hunk 1]
```diff
@@ -158,15 +132,87 @@ paths:
...
+              examples:
+                forward pagination:
+                  value:
+                    pagination:
+                      startCursor: 241de237-f989-42be-926d-c0c1fca57708
+                      endCursor: 315581f8-9967-45a0-9cd3-87b60b6d6414
+                      hasNextPage: true
+                      first: 10
+                    result:
+                      - id: 'https://openpayments.guide/alice/incoming-payments/016da9d5-c9a4-4c80-a354-86b915a04ff8'
```
[the remark this answers is not in the document]
Comment (sabineschaller) [reply open-api-spec.yaml:145]: It is a list with just one element. I can add a second to make it more clear.

[the remark this answers is not in the document]
Comment (wilsonianb) [reply open-api-spec.yaml:145]: oops, I wasn't wearing my yaml glasses :nerd_face:

**Review by wilsonianb — APPROVED** [review]

File: open-api-spec.yaml [hunk 2]
```diff
@@ -51,24 +37,22 @@ paths:
               schema:
                 $ref: '#/components/schemas/account'
               examples:
-                Get account for $openpayments.example:
+                Get account for $openpayments.guide/alice:
```
[the remark this answers is not in the document]
Comment (sabineschaller) [reply open-api-spec.yaml:40]: The [server URLs use `.guide`](https://github.com/interledger/open-payments/blob/f1df55b08afeb94224963f2d3769edf6f9460242/open-api-spec.yaml#L37) and I wanted to make it consistent. Could be that that even allows us to run the examples on readme.

File: open-api-spec.yaml [hunk 3]
```diff
@@ -158,15 +132,109 @@ paths:
...
           $ref: '#/components/responses/403'
       description: List all incoming payments on the account
-      parameters: []
+      parameters:
+        - schema:
+            type: integer
+            minimum: 1
+            maximum: 100
+            example: 10
+            default: 10
```
[the remark this answers is not in the document]
Comment (sabineschaller) [reply open-api-spec.yaml:219]: Happy to change that at a later point.

**Pull request merged**